File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.js
/**
 * @param {number[]} nums1
 * @param {number[]} nums2
 * @return {number[][]}
 */
var findDifference = function(nums1, nums2) {
    let commonList1 = [];
    let commonList2 = [];
    for(let i=0 ; i<nums1.length ; i++){
        if(!nums2.includes(nums1[i]) && !commonList1.includes(nums1[i])){
            commonList1.push(nums1[i]);
        }
    }
    for(let j=0;j<nums2.length;j++){
        if(!nums1.includes(nums2[j]) && !commonList2.includes(nums2[j])){
            commonList2.push(nums2[j]);
        }
    }
    return([commonList1,commonList2]);
};

File: 2215-find-the-difference-of-two-arrays/2215-find-the-difference-of-two-arrays.py
class Solution:
    def findDifference(self, nums1: List[int], nums2: List[int]) -> List[List[int]]:
        list1=[]
        list2=[]
        for i in nums1:
            if(i not in list1 and i not in nums2):
                list1.append(i)
        for j in nums2:
            if(j not in list2 and j not in nums1):
                list2.append(j)
        return[list1,list2]